### DISCUSSION TITLE
Run chained commands?

### DISCUSSION THREAD

**Discussion by alex-phillips** [bdd/runitor#18] [Q&A]
It doesn't look like it is working for me, but wanted to double check. Is it possible (or can it be a feature request) to pass chained commands into runitor? An example would be something like this:

```
runitor -uuid my-uuid -- cd /my/script/directory && ./my_script.js
```

I imagine We'd potentially have to surround the entire command string with quotes, but would be useful as these are some of the entries in my current crontab.

**Comment by bdd** [comment]
If you want to execute a shell statement, you need runitor to execute the shell for you. The passed command and its arguments are fed to `execv(3)`.

So in your use case, assuming your preferred shell is bash, you should run:
```
runitor -uuid my-uuid -- bash -c "cd /my/script/directory && ./my_script.js"
```

**Comment by alex-phillips** [comment]
D'oh, didn't even think to do that. Thanks!
